### PR TITLE
Improve wording of Config Upload password text

### PR DESCRIPTION
**Pull request by davidjb**

### CODE REVIEW

File: gui/templates/system/config_upload.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <tr>
   <td colspan="2">
    <p>{% trans "The system will reboot to perform this operation!" %}<p/>
-   <p style="color: red;">{% trans "All password will be reset in case your configuration bundle does not include the password secret seed." %}</p>
+   <p style="color: red;">{% trans "All passwords will be reset if your configuration bundle does not include the password secret seed." %}</p>
```
Comment (wblock) [inline gui/templates/system/config_upload.html]: I'd suggest s/your/the/

Is the rest of it clear enough?  Does the reader know what a "configuration bundle" is?

Comment (davidjb) [reply gui/templates/system/config_upload.html]: My fix was just the grammar whilst keeping the rest of the sentence the same. 

As for your other comments, the terminology ("configuration bundle") is slightly different than what the rest of the UI and docs say (http://doc.freenas.org/11/system.html#general) -- eg "Save Config", "Upload Config" in the UI but "configuration", "configuration database" and "configuration backup" are used equally in the docs.  

The issue runs deeper than just this one-line fix, but I'll adjust this for now.

**Comment by davidjb** [comment]
@wblock Try this new sentence.  `Configuration database` is used most commonly throughout various parts of the FreeNAS 11 docs (eg http://doc.freenas.org/11/system.html).

**Comment by wblock** [comment]
This is better, but implies that there is some way the reader can tell whether the password secret seed is in their saved configuration database.  How about:

If the uploaded configuration database file was saved without the Password Secret Seed, all passwords will be reset.

**Comment by davidjb** [comment]
Great suggestion @wblock, I've updated my PR accordingly.

On the topic of whether the config database was saved with/without the Seed, hopefully the admin who's restoring the content knows because they did it or can ask and find out.  Either way, this helpful text is about the best you can do given the circumstances and without rewriting all the logic for testing the configuration database.

**Pull request merged**